.github/ci-heapster.sh

```diff
@@ -46,7 +46,7 @@ setup_dist_bins() {
   fi
   extract_exe "saw" "dist/bin"
   export PATH=$PWD/dist/bin:$PATH
-  echo "$PWD/dist/bin" >> "$GITHUB_PATH"
+  echo "$PWD/dist/bin" >> $GITHUB_PATH
```

Comment:
Is this fixing the PATH issue we've previously had, or undoing the fix we put in place? I wonder if @ldettwy knows which we should be using.

Reply:
This appeared to be reverting a fix and I've added the quotes back.